sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/RestorePoint.cs

```diff
@@ -44,17 +50,14 @@ public RestorePoint()
         /// 'FileSystemConsistent', 'ApplicationConsistent'</param>
         /// <param name="provisioningDetails">Gets the provisioning details set
         /// by the server during Create restore point operation.</param>
-        /// <param name="excludeDisks">List of disk resource ids that the
-        /// customer wishes to exclude from the restore point. If no disks are
-        /// specified, all disks will be included.</param>
-        public RestorePoint(string id = default(string), string name = default(string), string type = default(string), RestorePointSourceMetadata sourceMetadata = default(RestorePointSourceMetadata), string provisioningState = default(string), string consistencyMode = default(string), RestorePointProvisioningDetails provisioningDetails = default(RestorePointProvisioningDetails), IList<ApiEntityReference> excludeDisks = default(IList<ApiEntityReference>))
+        public RestorePoint(string id = default(string), string name = default(string), string type = default(string), IList<ApiEntityReference> excludeDisks = default(IList<ApiEntityReference>), RestorePointSourceMetadata sourceMetadata = default(RestorePointSourceMetadata), string provisioningState = default(string), string consistencyMode = default(string), RestorePointProvisioningDetails provisioningDetails = default(RestorePointProvisioningDetails))
```

Comment:
1. this was generated automatically by the AutoRest. i did not intentionally change this
2. This PR (and the Swagger PR on which this PR is based) does have breaking changes. However, Jeffrey Richter and BreakingChanges Review Board team approved it (because no customers are currently using the api. this feature is still in  private preview)